.github/workflows/rich-code-nav.yml

```diff
@@ -0,0 +1,44 @@
+name: Rich Code Navigation
+
+on:
+  push:
+  pull_request:
```

Comment:
Maybe we shouldn't run this on PRs, but only when things are merged to main/stable, no?

Reply:
Left `push` only.